pkg/findprocess/findprocess_unix.go

```diff
@@ -10,7 +10,7 @@ import (
 func findProcess(pid int) (process *os.Process, err error) {
 	// On Unix systems, FindProcess always succeeds and returns a Process
 	// for the given pid, regardless of whether the process exists.
-	process, _ = os.FindProcess(pid)
+	process, _ = os.FindProcess(pid) // nolint: errcheck
```

Comment:
Shouldn't we treat this error? There's a potential nil dereference in the following line.

Reply:
Yeah I agree, I will check for the error there.